### DISCUSSION TITLE
M2 Ventura Subversion - svnserve command not found

### DISCUSSION THREAD

**Discussion by sonicnick** [Homebrew/discussions#4186] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 3.6.20-167-g7a5f614
ORIGIN: https://github.com/Homebrew/brew
HEAD: 7a5f6143d3e7447a64255c548d2e51138c83acd3
Last commit: 9 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 25e0de0968e7e3030405f96068380cff94dc292f
Core tap last commit: 8 days ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_INSTALL_FROM_API: set
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_blizzard_avalanche
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

I have subversion running under Ventura on an M2 Mac mini - I can check out local files and svnadmin commands are working. 
However when connecting from another machine eg:

`svn list svn+ssh://myuser@my.ip.addr/Users/myuser/myrepo`
I get the response:
```
zsh:1: command not found: svnserve
svn: E170013: Unable to connect to a repository at URL 'svn list svn+ssh://myuser@my.ip.addr/Users/myuser/myrepo'
svn: E210002: To better debug SSH connection problems, remove the -q option from 'ssh' in the [tunnels] section of your Subversion configuration file
svn: E210002: Network connection closed unexpectedly
```

Which seems to indicate that svnserve is not found in the paths. However my path seems to contain the relevant pointers

`/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin`

The repo was setup under the same user as the connection and has admin rights.
I can also call svnserve via ssh just not is seems via the svn+ssh connection

Now I'm stuck, and cant seem to make it work.

**Comment by cho-m** [comment]
Are you sure `PATH` is set up to what you want when `svn+ssh` is used? i.e. is it actually loading the corresponding shell config you want?

For example, on ZSH, `.zshrc` is loaded on interactive shells, `.zprofile` & `.zlogin` are loaded on login shells, `.zshenv` is loaded on all shells, etc.

**Reply by sonicnick** [reply]
Thanks, I think that got it.
